### PR TITLE
New release 2.1.7

### PR DESCRIPTION
**Pull request by mayanks**

### CODE REVIEW

File: bigquery/pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>com.google.cloud.bigdataoss</groupId>
     <artifactId>bigdataoss-parent</artifactId>
-    <version>2.1.7-SNAPSHOT</version>
```
Comment (mprashanthsagar) [inline bigquery/pom.xml:25]: why do we need to change this ?

**Comment by mprashanthsagar** [comment]
Please update the readme file

**Review by mprashanthsagar — CHANGES_REQUESTED** [review]

**Review by Deependra-Patel — APPROVED** [review]

**Review by mprashanthsagar — APPROVED** [review]

**Pull request merged**